tests/cli/test_rasa_test.py

```diff
@@ -148,42 +150,32 @@ def test_test_core_comparison(
 
 
 def test_test_core_comparison_after_train(
-    run_in_simple_project: Callable[..., RunResult]
+    run_in_simple_project: Callable[..., RunResult],
+    trained_rasa_model: str,
+    tmp_path: Path,
 ):
-    write_yaml(
-        {"language": "en", "policies": [{"name": "MemoizationPolicy"}]}, "config_1.yml"
-    )
 
-    write_yaml(
-        {"language": "en", "policies": [{"name": "MemoizationPolicy"}]}, "config_2.yml"
-    )
+    path = Path(tmp_path / "comparison_models")
+    path.mkdir()
+    assert path.exists()
 
-    run_in_simple_project(
-        "train",
-        "core",
-        "-c",
-        "config_1.yml",
-        "config_2.yml",
-        "--stories",
-        "data/stories.yml",
-        "--runs",
-        "2",
-        "--percentages",
-        "25",
-        "75",
-        "--out",
-        "comparison_models",
-    )
+    run_one = Path(path / "run_1")
+    run_one.mkdir()
+    assert run_one.exists()
+    shutil.copy(trained_rasa_model, run_one)
+
+    run_two = Path(path / "run_2")
+    run_two.mkdir()
+    assert run_two.exists()
+    shutil.copy(trained_rasa_model, run_two)
```

Comment:
optional: some tiny nit picks
```suggestion
    run_one = path / "run_1"
    run_one.mkdir()
    shutil.copy(trained_rasa_model, run_one)

    run_two = path / "run_2"
    run_two.mkdir()
    shutil.copy(trained_rasa_model, run_two)
```